src/ophyd_async/epics/areadetector/controllers/pilatus_controller.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+
+from ophyd_async.core import (
+    AsyncStatus,
+    DetectorControl,
+    DetectorTrigger,
+    set_and_wait_for_value,
+    wait_for_value,
+)
+
+from ..drivers.pilatus_driver import PilatusDriver, TriggerMode
+from ..utils import ImageMode
+
+TRIGGER_MODE = {
+    DetectorTrigger.internal: TriggerMode.internal,
+    DetectorTrigger.constant_gate: TriggerMode.ext_enable,
+    DetectorTrigger.variable_gate: TriggerMode.ext_enable,
+}
+
+
+class PilatusController(DetectorControl):
+    def __init__(self, drv: PilatusDriver) -> None:
+        self.driver = drv
+
+    async def get_deadtime(self, exposure: float) -> float:
+        return 0.002
```

Comment:
I got it wrong, should be
```suggestion
        return 0.001
```